acme/api/handler.go

```diff
@@ -359,6 +360,7 @@ func GetChallenge(w http.ResponseWriter, r *http.Request) {
 
 	w.Header().Add("Link", link(linker.GetLink(ctx, acme.AuthzLinkType, azID), "up"))
 	w.Header().Set("Location", linker.GetLink(ctx, acme.ChallengeLinkType, azID, ch.ID))
+	w.Header().Set("Cache-Control", "private, no-store")
```

Comment:
This is an ACME challenge object with a token.

Reply:
Will remove the header here per my response to the other comment about the challenge tokens.

Reply:
Removed the header here.